### PR TITLE
Analytics: add experimentview events

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What this PR does / why we need it**:
Add `experimentview` events to the RudderStack integration.

**Special notes for your reviewer**:
The first experiment is in the Enterprise repo (see associated PR), this is the reason why the helper function is not called in OSS. I wonder if I should add a comment somewhere to ensure this is not considered as unused and deleted by mistake in future versions.

### CODE REVIEW

File: packages/grafana-runtime/src/index.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export * from './services';
 export * from './config';
 export * from './types';
 export { loadPluginCss, SystemJS, PluginCssOptions } from './utils/plugin';
-export { reportMetaAnalytics, reportInteraction, reportPageview } from './utils/analytics';
+export { reportMetaAnalytics, reportInteraction, reportPageview, reportExperimentview } from './utils/analytics';
```
Comment (dprokop) [inline packages/grafana-runtime/src/index.ts:10]: That's an unusual selling, what about:
```suggestion
export { reportMetaAnalytics, reportInteraction, reportPageview, reportExperimentView } from './utils/analytics';
```

**Review by dprokop — APPROVED** [review]
Looks good, I'm only slightly sceptical about the spelling, but other than that 👍

File: packages/grafana-runtime/src/index.ts (hunk 1 above)
Comment (AgnesToulet) [reply packages/grafana-runtime/src/index.ts:10]: I agree, I just wanted to keep it consistent with `reportPageview`.

**Review by Clarity-89 — APPROVED** [review]
Looks good! Agree with the comment about the function naming.

**Pull request merged**